src/recipes/link.ts

```diff
@@ -0,0 +1,33 @@
+import { defineRecipe } from '@pandacss/dev'
+
+const linkRecipe = defineRecipe({
+  className: 'link',
+  description: '<a> styles',
+  base: {},
+  variants: {
+    in: {
+      modal: {
+        color: 'lightblue',
+      },
+      breadcrumbs: {
+        color: 'inherit',
+        textDecoration: 'none',
+      },
+      thoughtAnnotation: {
+        height: '1em',
+        display: 'inline-block',
+        overflow: 'hidden',
+        position: 'relative',
+        zIndex: 'thoughtAnnotationLink',
+      },
+      footer: {
+        color: { _dark: 'skyblue' },
+      },
+      tutorialText: {
+        color: { _dark: '#87ceeb' },
```

Comment:
Let's add `#87ceeb` as the global `_dark` color for `a` tags. Then we can remove the modal, footer, and tutorialText variants completely.

`skyblue` is identical to `#87ceeb`, and `lightblue` is so close that we should probably combine them.

Reply:
Ok sounds good.